imu_bno08x:
  ros__parameters:
    save_calibration: false
    publish_mag: false
    imu_frame: imu_link
    rate: 30
    address: 75
    frequency: 800000
